src/CC-BY-NC-SA-2.5.xml

```diff
@@ -0,0 +1,323 @@
+<spdx name="Creative Commons Attribution Non Commercial Share Alike 2.5" identifier="CC-BY-NC-SA-2.5" osi-approved="false">
+  <urls>
+    <url>http://creativecommons.org/licenses/by-nc-sa/2.5/legalcode</url>
+  </urls>
+  <license>
+    <title>
+      <p>Creative Commons Attribution-NonCommercial-ShareAlike 2.5</p>
+    </title>
+    <optional>
+      <p>CREATIVE COMMONS CORPORATION IS NOT A LAW FIRM AND DOES NOT PROVIDE LEGAL SERVICES. DISTRIBUTION OF THIS
+         LICENSE DOES NOT CREATE AN ATTORNEY-CLIENT RELATIONSHIP. CREATIVE COMMONS PROVIDES THIS INFORMATION ON
+         AN &quot;AS-IS&quot; BASIS. CREATIVE COMMONS MAKES NO WARRANTIES REGARDING THE INFORMATION PROVIDED,
+         AND DISCLAIMS LIABILITY FOR DAMAGES RESULTING FROM ITS USE.</p>
+      <p>License</p>
+    </optional>
+    <body>
+      <p>THE WORK (AS DEFINED BELOW) IS PROVIDED UNDER THE TERMS OF THIS CREATIVE COMMONS PUBLIC LICENSE
+         (&quot;CCPL&quot; OR &quot;LICENSE&quot;). THE WORK IS PROTECTED BY COPYRIGHT AND/OR OTHER APPLICABLE
+         LAW. ANY USE OF THE WORK OTHER THAN AS AUTHORIZED UNDER THIS LICENSE OR COPYRIGHT LAW IS
+         PROHIBITED.</p>
+      <p>BY EXERCISING ANY RIGHTS TO THE WORK PROVIDED HERE, YOU ACCEPT AND AGREE TO BE BOUND BY THE TERMS OF THIS
+         LICENSE. THE LICENSOR GRANTS YOU THE RIGHTS CONTAINED HERE IN CONSIDERATION OF YOUR ACCEPTANCE OF SUCH
+         TERMS AND CONDITIONS.</p>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>Definitions</p>
+        </li>
```

Comment:
should </li> be at end, rather than here. Brad and Sam think yes. need input from Kris